testgrid/config.yaml

```diff
@@ -1743,6 +1743,29 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-community-verify
   num_columns_recent: 20
 
+# kubernetes-sigs/release-notes
+- name: pull-release-notes-build-dev
```

Comment:
Sorry, I didn't realize you updated it! :woman_facepalming: 

I think this should be name of the job i.e. just `build-dev`.

Reply:
:innocent: I must be missing some detail somewhere int this file...